### ISSUE TITLE
Awesome plugin!

### ISSUE THREAD

**Issue by Zillew** [Norbo11/UltimateCards#2]
Awesome plugin, but the version I have is outdated, and I'm unnable to compile a new .jar using the files here. 
I don't suppose the compiled .jar is uploaded anywhere, really could do with updating! 

Thanks,
Zillew

**Comment by Norbo11** [comment]
Hi, thanks for your message.

The official Bukkit thread for the plugin is here: http://dev.bukkit.org/server-mods/ultimatecards/

You can find a compiled jar for the latest version there :)

I will, however, probably be uploading a new version either today or tomorrow so you might want to wait for that. Or enable the auto-update option ;)

Norbo

**Comment by Zillew** [comment]
Yeah, I was hoping to get a 1.8 compatible version, as the current version available on Bukkit is a bit glitchy :)

Also, side question...
When a player logs off, shouldn't they auto leave the table? Offline players are still sat at tables on my server, though perhaps it's because we're running old version...

**Comment by Norbo11** [comment]
The plugin should still work on 1.8 I believe. But I will be releasing an update very soon.

A player who logs off does not get removed from the table. This is because there are many things which could happen during a game which could cause a player to disconnect - computer crashes, internet downage, etc. I think players would be pretty frustrated to see their winnings thrown away due to a silly disconnect. That's why table owners have the choice of kicking players which do not come back into the game after leaving (/table kick [name]).

**Comment by Zillew** [comment]
The way I use your plugin is I guess pretty different, basically I have two tables set up (one high and one low) where players can play. I then used MyCommand to create a menu, so players choose which table and their buying just by clicking on buttons. (Saves them having to learn creation/joining commands)
Hence I was hoping for a feature like that... Perhaps you could implement an option in the config?

**Comment by Norbo11** [comment]
Hm, you're not the first person to use the plugin in such a way, and now that you point it out, an auto kick setting would be pretty useful. Will put it on my to-do list.

**Comment by Zillew** [comment]
Thanks for considering it! Could you poke me on Skype as soon as the update is available, so I can see it even if I'm on the go.
Username: Zillew

Thanks! :)
